docs/_partials-common/zephyr-setup-dependencies.md

```diff
@@ -54,11 +58,11 @@ If the Kitware repository doesn't work for you (ie: your system is not running U
 </TabItem>
 <TabItem value="macos">
 
-Start by installing dependencies with `brew`:
+1. Start by installing dependencies with `brew`:
```

Comment:
Is there a corresponding `2`?

Reply:
Not for macOS, but yes for the other two OSes so I figured it made sense to make this adhere to the formatting.